packages/react/src/patterns/sub-patterns/ContentGroup/ContentGroup.js

```diff
@@ -20,41 +21,44 @@ const { prefix } = settings;
  * @param {object} props props object
  * @param {*} props.children JSX Components
  * @param {string} props.heading  Heading objects
+ * @param {*} props.cta CTA component props object
  * @returns {*} JSX ContentArrayGroup component
  */
-const ContentGroup = ({ children, heading }) => {
-  return (
-    <div
-      className={`${prefix}--content-group`}
-      data-autoid={`${stablePrefix}--content-group`}>
-      <div className={`${prefix}--content-group__container`}>
-        <div className={`${prefix}--content-group__row`}>
-          <div className={`${prefix}--content-group__col`}>
-            <h3
-              data-autoid={`${stablePrefix}--content-group__title`}
-              className={`${prefix}--content-group__title`}>
-              {heading}
-            </h3>
-          </div>
+const ContentGroup = ({ children, heading, ...cta }) => (
+  <div
+    className={`${prefix}--content-group`}
+    data-autoid={`${stablePrefix}--content-group`}>
+    <div className={`${prefix}--content-group__container`}>
+      <div className={`${prefix}--content-group__row`}>
+        <div className={`${prefix}--content-group__col`}>
+          <h3
+            data-autoid={`${stablePrefix}--content-group__title`}
+            className={`${prefix}--content-group__title`}>
+            {heading}
+          </h3>
+        </div>
+      </div>
+      <div className={`${prefix}--content-group__row`}>
+        <div
+          data-autoid={`${stablePrefix}--content-group__children`}
+          className={classNames(
+            `${prefix}--content-group__col`,
+            `${prefix}--content-group__children`
+          )}>
+          {children}
         </div>
         <div className={`${prefix}--content-group__row`}>
```

Comment:
I don't see this doing anything.

Reply:
if row is not their CTA component is rendering inside the contentGroup placeholder
![image](https://user-images.githubusercontent.com/53789187/74067483-603ab080-49c7-11ea-8c69-094b452c432b.png)

Reply:
sorry if we remove `row` at 41 line .looks like above.

Reply:
but row is not doing anything at line number 50. will remove that.

Reply:
When I remove the grid I get the following. I think there might be another problem there.

<img src="https://user-images.githubusercontent.com/3793636/74067972-13071080-49c0-11ea-9bd8-6eef30b6452e.png" alt="" width="300px" />

Take a look at the rest of my feedback, it might resolve itself.

Reply:
where did you remove the grid?

Reply:
Here is what my `JSX` looks like after I removed the grid.

```jsx
<div
  className={`${prefix}--content-group`}
  data-autoid={`${stablePrefix}--content-group`}>
    <h3
      data-autoid={`${stablePrefix}--content-group__title`}
      className={`${prefix}--content-group__title`}>
        {heading}
    </h3>
    <div
      data-autoid={`${stablePrefix}--content-group__children`}
      className={classNames(
        `${prefix}--content-group__col`,
        `${prefix}--content-group__children`
      )}>
      {children}
    </div>
    <CTA style="card" type="local" {...cta} />
</div>
```

Looking in the `CSS` I'm not seeing those class names `__container`, `__row`, or `__col` set to anything.